apps/gimbtonomy/LEDMatrix.cpp

```diff
@@ -0,0 +1,47 @@
+/* LED Matrix Source File */
+
+#include "LEDMatrix.h"
+
+
+//Data constructor
+//LEDMatrix :: LEDMatrix (void)
```

Comment:
For the pin initialization in the constructor you may have to use the inline initialization. When I initialize pins for other peripherals I have had to do this but it may be different as you are using Pwmout pins as I was using Digital out pins. You can take a look at the Neopixel code if this ends up being the case. https://github.com/uwrobotics/MarsRover2020-firmware/blob/master/lib/neopixel/Neopixel_Blocking.cpp